dbsettings/values.py

```diff
@@ -63,6 +64,15 @@ def contribute_to_class(self, cls, attribute_name):
 
         setattr(cls, self.attribute_name, self)
 
+        if self.choices is not None:
+            # Allows for adding get_FIELD_display() to fields with choices.
+            if 'get_%s_display' % attribute_name not in cls.__dict__:
+                setattr(
+                    cls,
+                    'get_%s_display' % attribute_name,
```

Comment:
Do not compute `'get_%s_display' % attribute_name` twice.

Reply:
No problem.